test/utils/operator/orderFormatAmount.test.ts

```diff
@@ -0,0 +1,42 @@
+import BigNumber from 'bignumber.js'
+
+import { isTokenErc20, formattedAmount } from 'utils'
+
+import { TokenErc20 } from '@gnosis.pm/dex-js'
+
+const factoryToken = (): TokenErc20 => {
```

Comment:
Minor nitpick: any reason for not being a constant instead of a function?

Reply:
I was going to create a **TokenErc20** factory, for preference I wanted a concise syntax, however it is not necessary to make an object creator because it will not require variation.